src/components/Quiz/Quiz.tsx

```diff
@@ -0,0 +1,89 @@
+import './Quiz.css';
+
+const Quiz = () => {
+    const questionsObj = {
+        id: 0,
+        quiztitle: "Quiz Title 1",
+        questions: [
+            {
+                question: 'This is the first question.',
+                options: [
+                    'This is option 1',
+                    'This is option 2',
+                    'This is option 3',
+                    'This is option 4',
+                ],
+                answer: 'This is option 1',
+            },
+            // {
+            //     question: "Question 2",
+            //     options: [
+            //         'This is option 1',
+            //         'This is option 2',
+            //         'This is option 3',
+            //         'This is option 4',
+            //     ],
+            //     answer: "option2"
+            // },
```

Comment:
I guess you forgot to remove this comment.

Reply:
yes, this was just to test something